Documentation/ApiOverview/ContentSecurityPolicy/Index.rst

```diff
@@ -31,8 +35,17 @@ web page context.
 
     *   https://www.youtube.com/watch?v=a_cS2XfCplI&t=28766s
 
+CSP rules are used to describe, which external links or functionality are allowed for
```

Comment:
```suggestion
CSP rules are used to describe, which external assets or functionality are allowed for
```

links is for me something like `<a href>`, not a `<script src>`. Also it is also not only external. It is totally valid to disable, for example, font-src or form-action for self (can be set to "none").